### DISCUSSION TITLE
custom mbtiles doesn't show on map

### DISCUSSION THREAD

**Discussion by halukmy** [Mapsui/Mapsui#1990] [Q&A]
we have tried many Country from TTracktiler but doesnt show on map

**Comment by inforithmics** [comment]
Have you made a Sample Project, if yes I can have a look at it.

For Offline Mbtiles to work following steps need to be done.

1. Create Offline mbtiles (not vector only bitmap images are supported).
2. Configure the Map and copy the created mbtiles to the device. 
     The samples do this by extracting the mbtiles from the resource into the filesystem.
     Normally this is done by downloading the mbtiles to the device either by webservice or url.
3. Display the  map mbtiles
